hs/src/Reach/Connector/ALGO.hs

```diff
@@ -2343,6 +2343,10 @@ ce = \case
         -- [ extra ]
         return ()
     label after_lab
+  DLE_DataTag _ d -> do
+    ca d
+    cint 0
+    op "getbyte"
```

Comment:
Double check if this is really a number

Reply:
I'm not sure I understand this feedback.  Do you mean “double check that getbyte returns a number”?  If so, the docs say that it returns an integer.

Reply:
Yea. That's what I meant